test/test_builders.go

```diff
@@ -0,0 +1,156 @@
+///usr/bin/true; exec /usr/bin/env go run "$0" "$@"
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+func main() {
+	version := os.Args[1]
+
+	os.Setenv("FUNC_REGISTRY", "example.com/jdoe")
```

Comment:
What about using `gcr.io/boson-project` for this? I know that when just building, we don't push so this could be a dummy value. But I was working on some tests yesterday that are similar (less elegant as a shell script lol), and used this. I was thinking that maybe we could use GitHub's registry for upstream in the future instead of quay.io.

Reply:
`gcr.io` is GitHub's, not Google's?

Reply:
Yes. I have not pushed anything to it yet, but did set up my personal access token to do so.

Reply:
it's `ghcr.io` right?

Reply:
> But I was working on some tests yesterday that are similar (less elegant as a shell script lol)

I guess the two nice things about this script are that it uses `func` for the build, so actually tests using our code, and because of that you get some cheers along the way. :) 

```
>>>>>>>>>>>>>>> TESTING go-http
   🙌 Function image built: gcr.io/boson-project/go-http:latest
```